conf/requirements.txt

```diff
@@ -13,5 +13,5 @@ simplejson
 termcolor
 terminaltables
 yapf==0.31.0
-git+https://github.com/SymbiFlow/edalize.git@symbiflow_update#egg=edalize
+git+https://github.com/antmicro/edalize.git@f4pga#egg=edalize
```

Comment:
This should be reverted before merging, isn't it?

Reply:
Not exactly, we should still use updated edalize but I moved this branch to https://github.com/f4pga/edalize/tree/fpga-tool-perf.